### ISSUE TITLE
In high contrast (Black/White) mode, Visual focus indicator is properly not visible on "Learn more" link.

### ISSUE THREAD

**Issue by luckydennis** [microsoft/fluentui#13865]
### Environment Information

- **Package version(s)**: 7.116.0
- **Browser and OS versions**: Edge Chromium Version 83.0.478.56 and Windows 10

### Describe the issue:

In high contrast mode the visual focus is not visible for the link in Severe Warning and Warning single line message.

I did notice you can have multi-line= {true} but only have a single line message and the visual focus is visible on the link.

### Please provide a reproduction of the issue in a codepen:
1. turn on high contrast mode
2. go to https://developer.microsoft.com/en-us/fluentui#/controls/web/messagebar
3. tab to the link in the message bar, compare the visual focus for single line and multi line.

#### Actual behavior:

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/46434587/86161025-828db180-bac1-11ea-8558-fac32acec2b8.gif)

#### Expected behavior:

![ezgif com-video-to-gif (1)](https://user-images.githubusercontent.com/46434587/86161032-86b9cf00-bac1-11ea-8d7b-09ad4d5633ff.gif)

**Comment by xugao** [comment]
I think we should have a WindowText (not Highlight) color border when the link is focused. Similar to how `Link` component behaves.
@luckydennis - let me know if you disagree.

Note to dev: this issue is applicable for both Edge legacy & Chromium.

**Comment by luckydennis** [comment]
@xugao I am not sure what the color should be but I think the accessibility team just wants too see a visual focus.

**Comment by xugao** [comment]
@luckydennis - this actually looks like a bug with Edge in High Contrast mode. Few things I have observed:

1. it's working properly for some mode of the messagebar and some don't:
![image](https://user-images.githubusercontent.com/1207059/87705207-6bea8a00-c752-11ea-9626-8091113dc024.png)
Looks like for the broken case, browser is cutting off the `outline` for some reason.

2. If you turn off high contrast mode, with the exact same `outline` css, our component is working as expected. Therefore the browser is behaving differently when High contrast mode vs. normal mode.

I'm afraid there is not much we can do here. I will resolve this issue as external and report this bug to Edge team.

cc @tomi-msft

**Comment by luckydennis** [comment]
@xugao thank you for explanation 👍
